### ISSUE TITLE
Central widget without Titlebar

### ISSUE THREAD

**Issue by hulswit** [githubuser0xFFFF/Qt-Advanced-Docking-System#207]
Could you add an option to have a dock widget with no visible title bar? 
In my project I need a central dock widget that is not movable or closable and has no visible titlebar even if other dock widgets are visible. Using the  **HideSingleCentralWidgetTitleBar** solves this only in situation when the central dock widget is the only one visible. It would be great if a dock widget could be marked as not having title bar in all situations. May be this setting makes more sense for the containing CDockAreaWidget class?

**Comment by SleepProgger** [comment]
+1 for this.
The best i could come up with is:
```
ads::CDockAreaWidget *area = dock_manager->addDockWidget(ads::BottomDockWidgetArea, dock_widget);
area->setAllowedAreas(ads::OuterDockAreas);
area->titleBar()->setMaximumSize(0, 0);
```

For some reason `setVisible` did nothing even if it looks like that is exactly what ` HideSingleCentralWidgetTitleBar` does too.
Am a QT newb tho, so i might just have messed it up ;)

Would be grateful for any proper workaround too.

**Comment by SleepProgger** [comment]
Can be closed since #227 is merged, no ?

**Comment by githubuser0xFFFF** [comment]
@SleepProgger  Yes, thank you
